example/get_info.js

```diff
@@ -3,16 +3,16 @@ const fs   = require('fs');
 const ytdl = require('..');
 const id  = '7wNb0pHyGuI';
 
+const filepath = path.resolve(__dirname, 'info.json');
 
-ytdl.getInfo(id, function(err, info) {
+ytdl.getInfo(id, (err, info) => {
```

Comment:
not sure wether i should use promises here for sticking to new syntax or the callback since the promise is just a wrapper